app/controllers/userController.js

```diff
@@ -0,0 +1,40 @@
+import uuid from 'uuid/v4'
+import db from '../../database/models'
+import validateSignupData from '../../lib/helpers/validateSignupData'
+import encryptPassword from '../../lib/helpers/encrypt'
+
+class UserController {
+  static async signup(req, res) {
+    const errors = validateSignupData(req.body)
+    // eslint-disable-next-line no-console
+    console.log(errors)
```

Comment:
Kindly remove the `console.log` statement. as well as the comment above it.

Reply:
Okay